Gemfile

```diff
@@ -3,3 +3,5 @@ source 'https://rubygems.org/'
 # Work around hideousness of FakeFS
 require "pp"
 gemspec
+
+gem 'ox', '~> 2.4', '>= 2.4.1'
```

Comment:
I got this error in the Ruby 3.1 tests before I added an XML parser:
```
RuntimeError:
  Unable to find a compatible xml library. Ensure that you have installed or added to your Gemfile one of ox, oga, libxml, nokogiri or rexml
```

From this build: https://github.com/cipherstash/ruby-client/runs/7717614588?check_suite_focus=true#step:4:13.

I don't have a reason for picking `ox` over the other options in the list.

I assume that this isn't an issue in the ActiveStash tests because Rails already has an XML parser in its deps.